README.md

```diff
@@ -71,7 +71,7 @@ Oh God please yes! :heart: Feel free to look around the [<kbd>help wanted</kbd>]
 |                   | parser tests         | optimize tests       | emit tests         | parse                | desugar            | infer types        | optimize             | emit               |
 | ----------------- | -------------------- | -------------------- | ------------------ | -------------------- | ------------------ | ------------------ | -------------------- | ------------------ |
 | integers          | :heavy_check_mark:   | :warning: [[2]](#f2) | :heavy_check_mark: | :heavy_check_mark:   | :heavy_check_mark: | :heavy_check_mark: | :warning: [[2]](#f2) | :heavy_check_mark: |
-| floats            | :x: [[3]](#f3)       | :x: [[5]](#f5)       | :x: [[3]](#f3)     | :x: [[4]](#f4)       | :x: [[3]](#f3)     | :x: [[3]](#f3)     | :x: [[5]](#f5)       | :x: [[3]](#f3)     |
+| floats            | :heavy_check_mark:   | :x: [[5]](#f5)       | :x: [[3]](#f3)     | :heavy_check_mark:   | :x: [[3]](#f3)     | :x: [[3]](#f3)     | :x: [[5]](#f5)       | :heavy_check_mark: |
```

Comment:
emit tests can be :heavy_check_mark: instead of :x: here

Reply:
This is what I meant:
![emit](https://user-images.githubusercontent.com/149425/60805329-0d262400-a180-11e9-8002-a9e237781e6d.png)

btw I think **we can :heavy_check_mark: everything except optimizations** in that column, and under the table make the `f3` and `f4` anchors similar to `f6` (recycle them: remove the note there but keep the footnote existing with something like `this space left intentionally blank` so that we don't have to cascade number changes into the table)

If my description of this is confusing (English is not my mother language either), tell me and I'll just do it myself :sweat_smile: #languageBarriers